shotover/src/transforms/query_counter.rs

```diff
@@ -16,6 +18,7 @@ use super::UpChainProtocol;
 #[derive(Clone)]
 pub struct QueryCounter {
     counter_name: &'static str,
+    counters: HashMap<String, Counter>,
```

Comment:
Can we name this something like `query_to_counter` to indicate the relation between the key and value?
I find this especially important when we have a string or integer type as the key, since in that case its not clear what kind of value the key actually is.